Sources/MapboxMaps/Annotations/ViewAnnotation.swift

```diff
@@ -86,24 +90,25 @@ public final class ViewAnnotationManager {
         viewsById[id] = view
         idsByView[view] = id
         expectedHiddenByView[view] = !(creationOptions.visible ?? true)
+        creationOptions.associatedFeatureId.flatMap { viewsByFeatureIds[$0] = view }
```

Comment:
I don't think I'd use flatMap just for its side-effects. `if let` is more appropriate in this situation.